parallel-install/pkg/overrides/overrides_test.go

```diff
@@ -113,27 +119,4 @@ func Test_ReadOverridesFromCluster(t *testing.T) {
 		require.Equal(t, 2, len(res2["global"].(map[string]interface{})), "Number of global overrides not as expected")
 		require.Equal(t, "changed", globalOverrides["globalOverride1"], "Override from additional overrides not on top of regular overrides")
 	})
-
```

Comment:
Removed this test-function as multiple overlay-files will be handled by the `deployment.Overrides` now. This test-case is no longer needed.